packages/phpstan-rules/tests/Rules/CheckTypehintCallerTypeRule/CheckTypehintCallerTypeRuleTest.php

```diff
@@ -29,6 +29,7 @@ public function provideData(): Iterator
         yield [__DIR__ . '/Fixture/SkipMayOverrideArg.php', []];
         yield [__DIR__ . '/Fixture/SkipMultipleUsed.php', []];
         yield [__DIR__ . '/Fixture/SkipNotPrivate.php', []];
+        yield ['vendor/rector/rector/packages/better-php-doc-parser/src/ValueObject/PhpDocNode/Doctrine/AbstractDoctrineTagValueNode.php', []];
```

Comment:
@TomasVotruba I tried add the rector abstract class that shown error at rector repo, but seems can't reproduce it